kbase-extension/static/kbase/js/widgets/appWidgets2/input/selectInput.js

```diff
@@ -221,6 +216,19 @@ define([
 
         // LIFECYCLE API
 
+        function buildAllOptions() {
+            return model.availableValues.map((item) => buildOption(item));
+        }
+
+        function buildOption(item = {}) {
+            return {
+                value: item.value,
+                id: item.value,
+                text: item.display,
+                disabled: model.disabledValues.has(item.value),
+            };
```

Comment:
do you need the extra function here? I find it just as easy to parse
```js
   return model.availableValues.map((item) => {
      return {
         value: item.value,
         ...
      }
   })
```

but that might just be me!

Reply:
Yeah, `buildOption` gets used elsewhere as a convenience function in the case where we want to start up with a single item already selected.